tensorboard/plugins/mesh/tf_mesh_dashboard/mesh-viewer.ts

```diff
@@ -411,11 +428,15 @@ export class MeshViewer extends THREE.EventDispatcher {
     // Make sure that even after arbitrary rotation mesh won't be clipped.
     const camera_to_far_edge = min_z < 0 ? -min_z + camera_z : camera_z - min_z;
     // Set camera position and orientation.
-    this.setCameraViewpoint(
-      {x: center.x, y: center.y, z: camera_z},
-      camera_to_far_edge * 3,
-      center
-    );
+    let camera_position = new THREE.Vector3(center.x, center.y, camera_z);
+    if (this._init_camera_position) {
+      camera_position = this._init_camera_position;
+    }
```

Comment:
FYI, this can be rewritten as: 
```ts
const cameraPosition = this._init_camera_position ?? new THREE.Vector3(center.x, center.y, camera_z);
```
This syntactic sugar is not required.